src/components/CartOrderForm.js

```diff
@@ -111,14 +116,36 @@ export const CartOrderForm = ({
             inputProps={user.userType === 'user' ? cartFormClasses.dateProps : {}}
           />
 
-          <FormControl align='left' fullWidth={true} error={deliveryMethodError} sx={{ my: 1 }}>
-            <FormLabel id='demo-row-radio-buttons-group-label'>Método de entrega</FormLabel>
-            <RadioGroup row aria-labelledby='demo-row-radio-buttons-group-label' name='row-radio-buttons-group' onChange={handleRadioClick}>
-              <FormControlLabel value='delivery' control={<Radio />} label='Entrega' checked={deliveryMethod === 'delivery'} />
-              <FormControlLabel value='takeAway' control={<Radio />} label='Take Away' checked={deliveryMethod === 'takeAway'} />
-            </RadioGroup>
-          </FormControl>
+          <Box sx={{ display: { xs: 'block', md: 'flex' }, mb: 2 }}>
+            <FormControl align='left' fullWidth={true} error={deliveryMethodError} sx={{ my: 1 }}>
+              <FormLabel id='demo-row-radio-buttons-group-label'>Método de entrega</FormLabel>
+              <RadioGroup row aria-labelledby='demo-row-radio-buttons-group-label' name='row-radio-buttons-group' onChange={handleRadioClick}>
+                <FormControlLabel value='delivery' control={<Radio />} label='Entrega' checked={deliveryMethod === 'delivery'} />
+                <FormControlLabel value='takeAway' control={<Radio />} label='Take Away' checked={deliveryMethod === 'takeAway'} />
+              </RadioGroup>
+            </FormControl>
 
+            {deliveryMethod === 'delivery' && user.userType === 'admin' && (
+              <FormControlLabel
+                control={<Switch checked={haveExtraFee} onChange={() => setHaveExtraFee(!haveExtraFee)} inputProps={{ 'aria-label': 'controlled' }} />}
```

Comment:
what does `aria-label: controlled` do?

Reply:
That is for accessibility when the switch does not have a label. Ill remove it cause we have a label.